shaders/IronChunk.gdshader

```diff
@@ -1,30 +1,42 @@
 shader_type spatial;
 render_mode depth_prepass_alpha;
 
-uniform sampler2D albedoTexture : source_color;
+uniform sampler2D fullTexture : source_color;
+uniform sampler2D depletedTexture : source_color;
 uniform sampler2D normalTexture;
 
+// Increase to fade the iron out of the rock
+uniform float depletion : hint_range(0, 1);
+
 uniform sampler2D dissolveTexture : source_color;
 uniform float dissolveValue : hint_range(0, 1);
-
 uniform float outlineWidth;
 uniform vec4 growColor : source_color;
 
 void fragment() {
-    vec4 mainTex = texture(albedoTexture, UV);
+    vec4 fullTex = texture(fullTexture, UV);
+    vec4 depletedTex = texture(depletedTexture, UV);
+    vec3 mixcolor = (fullTex.rgb * (1.f - depletion)) + (depletedTex.rgb * depletion);
+    
+    //float advance = clamp(depletion + sin(TIME / 1.f), 0, 1);
+    
+    //vec3 mixcolor = (fullTex.rgb * (1.f - advance)) + (depletedTex.rgb * advance);
```

Comment:
Should these  two commented out lines be removed? I'm not sure what these are related to (and if it would make any sense to keep these).